build_tools/circle/build_doc.sh

```diff
@@ -123,7 +123,7 @@ conda create -n $CONDA_ENV_NAME --yes --quiet python="${PYTHON_VERSION:-*}" \
   numpy="${NUMPY_VERSION:-*}" scipy="${SCIPY_VERSION:-*}" cython \
   pytest coverage matplotlib="${MATPLOTLIB_VERSION:-*}" sphinx=1.6.2 pillow \
   scikit-image="${SCIKIT_IMAGE_VERSION:-*}" pandas="${PANDAS_VERSION:-*}" \
-  joblib
+  joblib nomkl
```

Comment:
will this slow down the example gallery?

Reply:
Let me re-trigger a full docs build to double check, but as far as I know, MKL and OpenBLAS should have a roughly comparable performance for linear algebra. The latter is ~150-200 MB smaller though, so faster to install with conda.

Reply:
In the end, I reverted this change as there were a few convergence issues that weren't there with MKL.